### PR TITLE
feat: Profile View

### LINKED ISSUE #156: Create profile view

### PR DESCRIPTION
**Pull request by GeorginaFraire**
#### What does this PR do?
- Add a new view to display user information such as questions they asked, commented, and in the case of the admin, questions they answered.

#### Where should the reviewer start?
- app/routes/profile.jsx
- app/components/QuestionTable
- app/components/Atoms/Table

#### How should this be manually tested?
1. Login
2. Go to the user image, open the menu and click on the profile picture
4. :tada:

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #156 

https://user-images.githubusercontent.com/97203617/225450669-78ea8616-4ed5-4191-bfc1-b7e8c00359b8.mov

### CODE REVIEW

**Review by albertcastaned — COMMENTED** [review]
Unless it's intentional, I think maybe we should make the profile access more obvious for users?. It could be hidden unless someone clicks on their own image by accident. Maybe adding a new button in the profile dropdown?

**Comment by GeorginaFraire** [comment]
> Unless it's intentional, I think maybe we should make the profile access more obvious for users?. It could be hidden unless someone clicks on their own image by accident. Maybe adding a new button in the profile dropdown?

Yeah! I will add a new option in the navbar.

File: app/controllers/profile/questionsCreated.js [hunk 1]
```diff
@@ -20,12 +20,43 @@ const getQuestionsCreated = async (query) => {
...
+            Votes: true,
+          },
+        },
+        Votes: true,
+        Department: true,
+      },
+    });
+
+    const questions = fetchQuestions.map((question) => {
+    // eslint-disable-next-line array-callback-return, consistent-return
```
Comment (albertcastaned) [inline app/controllers/profile/questionsCreated.js:40]: If possible, lets try to avoid eslint disable overrides to mantain consistency

**Review by albertcastaned — COMMENTED** [review]
@GeorginaFraire Continued reviewing, left some more comments , will test locally next.

**Review by albertcastaned — APPROVED** [review]
LGTM 🎉

**Review by Carlos-Reyna — APPROVED** [review]

**Pull request closed without merging**